Tests/Pcap++Test/TestDefinition.h

```diff
@@ -65,6 +65,8 @@ PTF_TEST_CASE(TestTcpReassemblyIPv6_OOO);
 PTF_TEST_CASE(TestTcpReassemblyCleanup);
 PTF_TEST_CASE(TestTcpReassemblyMaxOOOFrags);
 PTF_TEST_CASE(TestTcpReassemblyMaxSeq);
+PTF_TEST_CASE(TestTcpReassemblyDBOOOBufferCC);
```

Comment:
Too many initials 😄 
what do you think about: `TestTcpReassemblyDisableOOOCleanup`